index.js

```diff
@@ -16,7 +16,7 @@ const fs = require('fs-extra');
 const program = require('commander');
 
 const commands = require('./commands/index.js');
-const pkg = require('../package.json');
+const pkg = require('./package.json');
```

Comment:
oops, I had a `package.json` in the parent directory, which allowed this to not fail for me. good catch. thanks!